README.md

```diff
@@ -0,0 +1,188 @@
+img src="https://raw.githubusercontent.com/hpe-design/logos/master/Requirements/color-logo.png" alt="HPE Logo" height="100"/>
+
+# RAG demo (Chat with HPE Press Release version)
+
+<b>Author:</b> Tyler Britten, Andrew Mendez </br>
+<b>Date:</b> 05/01/2024</br>
+<b>Revision:</b> 0.1</br>
+
+This demonstration was built to showcase Retrieval Augmented Generation (RAG) on HPE press release documents.
+It shows how RAG can be used to assist customers with keeping up with recent HPE news articles.
+
+To replicate this demo, you will need:
+
+ - A functioning Kubernetes cluster with load balancers for external facing services configured
+     - cluster having shared mounted folder `/nvmefs1/tyler.britten`
+ - Pachyderm/HPE MLDM 2.9.2 installed on the cluster and fully functional
+ - At least 1x NVIDIA A100 80GB GPUs
+ - Determined.AI/HPE MLDE environment for finetuning models (not included in the base code here)
+ - 
+
+<b>NOTE:</b> You might be able to replicate this demo with other GPUs (for example L40s) as well, but you need to consider the memory footprint of other GPUs and adjust accordingly.
+
+
+## Recorded demos of RAG demo
+
+[ToDo]
+
+## Implementation Overview
+The below chart gives a logical overview of the application flow of Mercedes Me Chat. </br>
```

Comment:
Mercedes

Reply:
lol bad copying and pasting here, will remove this.

Reply:
removed